model/arrayWrapper.js

```diff
@@ -2,12 +2,26 @@
  * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
  * See LICENSE for license details.
  */
-export function arrayToObject(arr, keyField = 'id') {
-    return { ...arr.map(item => ({ [keyField]: item })) };
+export function maxObjectValueInArrayByObjectKey(array, propName) {
+    if (array.length < 1) return 0;
+
+    return array.reduce(
+        (prev, current) => {
+            if (!current || !current[propName]) return prev;
+            return (prev[propName] > current[propName] ? prev : current);
+        },
+    )[propName];
 }
 
-export function sumArray(arr) {
-    return arr.reduce((a, b) => a + b, 0);
+export function minObjectValueInArrayByObjectKey(array, propName) {
+    if (array.length < 1) return 0;
+
+    return array.reduce(
+        (prev, current) => {
+            if (!current || !current[propName]) return prev;
+            return (prev[propName] < current[propName] ? prev : current);
+        },
```

Comment:
pass initial value to inform what kind of data structure should be returned

Reply:
I don't get it - the initial data ( method arguments ) won't match returning value

Reply:
Yup, you are right, becouse it's objects comparison.